### ISSUE TITLE
Failure to load configuration method

### ISSUE THREAD

**Issue by SXRWahrheit** [greatman/craftconomy3#110]
```
[11:12:07] [Server thread/INFO]: [Craftconomy3] Enabling Craftconomy3 v3.3-e72ac42-trv178
[11:12:07] [Server thread/ERROR]: Error occurred while enabling Craftconomy3 v3.3-e72ac42-trv178 (Is it up to date?)
java.lang.NoSuchMethodError: org.bukkit.configuration.file.YamlConfiguration.loadConfiguration(Ljava/io/InputStream;)Lorg/bukkit/configuration/file/YamlConfiguration;
	at com.greatmancode.craftconomy3.tools.configuration.bukkit.BukkitConfig.<init>(BukkitConfig.java:59) ~[?:?]
	at com.greatmancode.craftconomy3.tools.interfaces.BukkitLoader.onEnable(BukkitLoader.java:43) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[server.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:402) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:377) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:327) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:421) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:382) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:337) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:272) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:544) [server.jar:git-Spigot-ede465d-bb4ae3b]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
```

**Comment by Proplex** [comment]
This is happening to me as well after updating to 1.12

**Comment by greatman** [comment]
I did the fix here: https://github.com/greatman/GreatmancodeTools/commit/c785e4f878e335cddd65c31afa92751be094de41

Travis is being a pain right now so it's not being built. Should work tho if you compile yourself (build/publish in gradle to your local repo) and then compile craftconomy.

**Comment by greatman** [comment]
Nevermind, it worked finally, you can find the updated version here: https://oss.sonatype.org/content/repositories/snapshots/com/greatmancode/craftconomy3/3.3.1-SNAPSHOT/craftconomy3-3.3.1-20170608.213153-17.jar

**Comment by Proplex** [comment]
Just compiled it myself, and tried your .jar as well.

This time, it's outputting: 
```[17:36:44] [Server thread/ERROR]: Error occurred while enabling Craftconomy3 vcustombuild (Is it up to date?)
java.lang.IllegalArgumentException: URI is not hierarchical
	at java.io.File.<init>(File.java:418) ~[?:1.8.0_131]
	at com.greatmancode.craftconomy3.tools.configuration.bukkit.BukkitConfig.<init>(BukkitConfig.java:68) ~[?:?]
	at com.greatmancode.craftconomy3.tools.interfaces.BukkitLoader.onEnable(BukkitLoader.java:46) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:402) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:377) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:327) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:421) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:382) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:337) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:272) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:544) [spigot.jar:git-Spigot-ede465d-bb4ae3b]
````

**Comment by greatman** [comment]
Changed back to using the old method, but the new function of loading Yaml files with Readers: https://oss.sonatype.org/content/repositories/snapshots/com/greatmancode/craftconomy3/3.3.1-SNAPSHOT/craftconomy3-3.3.1-20170608.230020-18.jar

Should have better chance!

**Comment by SXRWahrheit** [comment]
This newest version appears to load properly, thanks. Will give it some more thorough testing once my other plugins have updated.

**Comment by greatman** [comment]
I just published 3.3.1 with the 1.12 update:
https://dev.bukkit.org/projects/craftconomy/files/2442635

Still under review.
